examples/SecuredTwoWay-Esp8266-Client/SecuredTwoWay-Esp8266-Client.ino

```diff
@@ -35,29 +35,6 @@ const char *ntp1 = "time.windows.com";
 const char *ntp2 = "pool.ntp.org";
 time_t now;
 
-// The hardcoded certificate authority for this example.
-// Don't use it on your own apps!!!!!
-const char ca_cert[] PROGMEM = R"EOF(
```

Comment:
Why did you replace the cert here with fingerprint? I think I rather have this example as is (even if the certificate is outdated, to show usage of `setTrustAnchors`)

This doesn't work without changing the certificate?

Reply:
Already restore the original example.

Reply:
Just updated the CA Cert for `SecuredTwoWay-Esp8266-Client` example